src/Symfony/Bridge/Twig/Node/SearchAndRenderBlockNode.php

```diff
@@ -107,4 +113,24 @@ public function compile(Compiler $compiler): void
 
         $compiler->raw(')');
     }
+
+    /**
+     * @see \Twig\Extension\CoreExtension::testEmpty()
+     */
+    private static function isEmpty($value): bool
```

Comment:
If we rely on the internal twig function for the compiled template, we can do the same for the compilation step and avoid duplicating the code.

But I'm still not sure this is a sane solution to rely on the internal function.

Reply:
I agree, let's use the Twig method instead.

Reply:
Maybe these methods shouldn't be `@internal` then?